binaries/scootapi/main.go

```diff
@@ -11,7 +11,7 @@ func main() {
 	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
 	transportFactory := thrift.NewTTransportFactory()
 
-	dialer := client.NewDialer("localhost:9090", transportFactory, protocolFactory)
+	dialer := client.NewDialer(transportFactory, protocolFactory)
```

Comment:
My concern is that it'd be easy to think this is the default then, but then it gets overwritten by the default in the flag. WDYT?

(BTW, fwiw, overall I think this is kinda shady all around; I'm not super happy with how this is working out. Maybe just get rid of dialer altogether and just have it baked into the client object instead of trying to separate them?)